api/config/featured_workspaces_test.json

```diff
@@ -1,9 +1,22 @@
 {
   "featuredWorkspaces": [
     {
-      "name": "wharrgarbl",
-      "namespace": "aou-rw-test-ebb1d76d",
-      "id": "wharrgarbl"
```

Comment:
This workspace apparently does exist.  But my user can't see it, so maybe it's not Published?  anyways, let's remove it from here.